go/http/api.go

```diff
@@ -363,6 +363,10 @@ func (this *HttpAPI) PostCopy(params martini.Params, r render.Render, req *http.
 	if err := this.validateToken(r, req); err != nil {
 		return
 	}
+	qs := req.URL.Query()
+	if sourceHost, exists := params["sourceHost"]; !exists || sourceHost=="" {
```

Comment:
In `golang`, if the value does not exist in the map, then the returned value is `""` anyway. So you can rewrite as:

```go
if sourceHost := params["sourceHost"]; sourceHost == "" {
```